### ISSUE TITLE
How to filter with comparison operators

### ISSUE THREAD

**Issue by aronhamvas** [restsql/restsql#16]
I was wondering if it is possible to use '>' and the other comparison operators. So far, the only way I found was SQL injection. :)

Example:
http://localhost:8080/restsqltest/res/User?USER_NAME=123'%20OR%20ID%20%3E%2010%20AND%20'a'%20=%20'a
saying USER_NAME='123' OR ID > 20 AND 'a' = 'a' to give me users with ids > 20.

**Comment by restsql** [comment]
The notification link takes me to #16, which was closed, but I don't see
your comment. So I am replying via email.

It does look like an error. I glanced through the code and tests and it
always passes in Equals when using that second constructor. It's puzzling
why the comment says it sets it to equals when it can just pass in the
value. I believe HTTP paths are using the first ctor, which parses the
operator from the parameter value. Are you using the Java API?

I noticed you forked the project. Did you try changing second ctor and it
works for you?

Thanks,
Mark
